packages/evo-marko/src/tags/evo-ccd/index.marko

```diff
@@ -52,7 +53,7 @@ export interface Input extends Marko.HTML.Div {
             <evo-icon-ccd-top class="ccd__top-icon"/>
             <div class="ccd__body">
                 <div>${min} - ${max}</div>
-                <div>W</div>
+                <div>${units}</div>
```

Comment:
Seems like `<evo-ccd>` doesn't have all features from `<ebay-ccd>`, I went ahead and added this `units` since I didn't want to keep an unused type but am happy to revert